src/service/reporting/reportService.ts

```diff
@@ -64,14 +64,26 @@ Promise<{ csvFile: JobsFile }> => {
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-expect-error
   const emailIds = (await tableService.getJobData('skillsSync', 'emailIds')).split(',')
-  let csvFilenamePrefix
+  let csvFilenamePrefix: string
+  const lastFile = {
+    lastFileOperation: await tableService.getJobData('skillsSync', 'lastFileOperation'),
+    lastFileDate: await tableService.getJobData('skillsSync', 'lastFileDate'),
+    lastFileSequenceNumber: await tableService.getJobData('skillsSync', 'lastFileSequenceNumber')
+  }
```

Comment:
Can we extract this object as a type? Example:

```ts
type SkillsCompletedLearnerRecordsFileDetails = {
    operation: "create" | "update" | undefined,
    date: string | undefined,
    sequenceNumber: string | number | undefined
}
```
Then we can implement the `lastFile` code like so:

```ts
const lastFile: SkillsCompletedLearnerRecordsFileDetails = {
    operation:  await tableService.getJobData('skillsSync', 'lastFileOperation'),
    date: await tableService.getJobData('skillsSync', 'lastFileDate'),
    sequenceNumber: await tableService.getJobData('skillsSync', 'lastFileSequenceNumber')
}
```